packages/py3_pip.rb

```diff
@@ -2,7 +2,7 @@
 
 class Py3_pip < Python
   description 'Pip is the python package manager from the Python Packaging Authority.'
-  homepage 'https://pip.pypa.io/'
+  homepage 'https://pypi.python.org/pypi/pip'
```

Comment:
`https://pypi.org/project/pip`?

Reply:
But this url redirects to https://pypi.org/project/pip?

Reply:
Reverted the change.